release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 18. | **Feature** | Add Central Secret Store(CSS) to SystemServicesForScale | **Brief Description**: Instantiate the system service Central Secret Store (CSS) with good defaults, omitting the need for additional user-provided configuration settings.<br>**Impact**: Users previously had to specify configuration settings upon activating the Central Secret Store service; failing to specify these settings led to cluster upgrade failures. After this change, enabling the service proceeds with safe defaults for these settings.<br>**Solution**: Add Central Secret Store to the SystemServicesForScale sections of WRPSettings.xml in the WindowsFabric repository, and to Settings.xml in the winfab_rp repository.<br>**Documentation Reference**: Read more about it at -<ul><li>[Enable Central Secrets Service](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-application-secret-store#enable-central-secrets-service)</li></ul>**Breaking Change**: Attempting to manually set the MinReplicaCount and TargetReplicaCount configuration settings of the Central Secret Store service will be overridden with default values as determined by the runtime for any system service.<br>**Workaround**: [WORKAROUND] |
+| 19. | **Feature** | Support adding Service Fabric Resource Provider (SFRP) metadata to app type + version entities, starting with ARM resource Id | **Brief Description**: Added a new API that can be used to add SFRP metadata to app type entities.<br>**Impact**: To better support SFRP & SFMC scenarios, SF runtime needs to allow support for additional metadata for ARM deployed resources. |
+| 20. | **Bug Fix** | Managed Identity Token Service (MITS) fails renegotiation with gateway on systems with TLS 1.3 enabled | **Brief Description**: The Managed Identity Token Service (MITS) will lose its connection to the gateway when deployed on Win2022, and won't attempt to renegotiate it.<br>**Impact**: On Windows 2022 or other situations where TLS1.3 is available/the default, MITS will fail to renegotiate its inbound connections, and will fail all inbound requests. SF-hosted user services attempting to acquire tokens representing their managed identities will experience timeouts.<br>**Fix**: Limit MITS to using TLS 1.2 only, until Windows 2022 becomes a supported OS for Service Fabric.<br>**Workaround**: Note Service Fabric does not list Windows 2022 as a supported OS version at this time - primarily because it defaults to TLS1.3, and the SF runtime needs additional fixes to support this scenario. Our recommendation is to stick with the officially supported OS images/versions. In this particular case, restarting MITS on the failing node will work momentarily, which may be sufficient for an application to obtain its tokens. However, the failing behavior will re-occur within the order of minutes. |
```

Comment:
```suggestion
| 20. | **Bug Fix** | Managed Identity Token Service (MITS) fails renegotiation with gateway on systems with TLS 1.3 enabled | **Brief Description**: MITS will lose its connection to the gateway when deployed on Win2022 and won't attempt to renegotiate it.<br>**Impact**: On Windows 2022 or other situations where TLS1.3 is available/the default, MITS will fail to renegotiate its inbound connections and will fail all inbound requests. SF-hosted user services attempting to acquire tokens representing their managed identities will experience timeouts.<br>**Fix**: Limit MITS to using TLS 1.2 only, until Windows 2022 becomes a supported OS for Service Fabric.<br>**Workaround**: Note Service Fabric does not list Windows 2022 as a supported OS version at this time - primarily because it defaults to TLS1.3, and the SF runtime needs additional fixes to support this scenario. Our recommendation is to stick with the officially supported OS images/versions. In this particular case, restarting MITS on the failing node will work momentarily, which may be sufficient for an application to obtain its tokens. However, the failing behavior will reoccur within the order of minutes.
```

Reply:
^replaced second instance of Managed Identity Token Service with MITS; removed unnecessary commas; removed unnecessary hyphen